lib/ansible/parsing/vault/__init__.py

```diff
@@ -736,7 +736,7 @@ def decrypt_and_get_vault_id(self, vaulttext, filename=None):
                     vault_secret_used = vault_secret
                     file_slug = ''
                     if filename:
-                        file_slug = ' of "%s"' % filename
+                        file_slug = u' of "%s"' % to_text(filename)
```

Comment:
I **think** you need to ensure that you are not merging bytes and unicode in `display.vvvv` below as well; use `u""` and make sure that all args are unicode. Also, line 737 needs `file_slug = u''`?